backend_skeleton/front.py

```diff
@@ -0,0 +1,40 @@
+# This script will act as a front to communicate with central.
+
+import socket
+import time
+
+def client():
+    client_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    server_ip = 'localhost'
+    server_port = 17400
+    server_address = (server_ip, server_port)
+    client_socket.connect(server_address)
+    print("Backend Reached Succesfuly!")
+    try:
+        while True:
+            print("Type 'quit' to quit.")
+            data = client_socket.recv(1024)
+            print(f"Central says: {data.decode('utf-8')}")
```

Comment:
I would decode earlier:
```suggestion
            data = client_socket.recv(1024).decode('utf-8')
            print('Central says:', data)
```